<html>
<head>
<title>TestNG:  SecondTest</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>SecondTest</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>4/2/0</td>
</tr><tr>
<td>Started on:</td><td>Fri Mar 16 12:14:18 MSK 2018</td>
</tr>
<tr><td>Total time:</td><td>0 seconds (5 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.epam.test.calculator.NegativTests.testSin()'><b>testSin</b><br>Test class: com.epam.test.calculator.NegativTests</td>
<td><div><pre>java.lang.AssertionError: expected [false] but found [true]
	at com.epam.test.calculator.NegativTests.testSin(NegativTests.java:38)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace2057458326", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace2057458326'><pre>java.lang.AssertionError: expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertFalse(Assert.java:65)
	at org.testng.Assert.assertFalse(Assert.java:75)
	at com.epam.test.calculator.NegativTests.testSin(NegativTests.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
</pre></div></td>
<td>0</td>
<td>com.epam.test.calculator.NegativTests@7ff95560</td></tr>
<tr>
<td title='com.epam.test.calculator.NegativTests.testSinNeg()'><b>testSinNeg</b><br>Test class: com.epam.test.calculator.NegativTests</td>
<td><div><pre>java.lang.AssertionError: expected [false] but found [true]
	at com.epam.test.calculator.NegativTests.testSinNeg(NegativTests.java:30)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace343264596", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace343264596'><pre>java.lang.AssertionError: expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertFalse(Assert.java:65)
	at org.testng.Assert.assertFalse(Assert.java:75)
	at com.epam.test.calculator.NegativTests.testSinNeg(NegativTests.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
</pre></div></td>
<td>0</td>
<td>com.epam.test.calculator.NegativTests@7ff95560</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.epam.test.calculator.NegativTests.testCos()'><b>testCos</b><br>Test class: com.epam.test.calculator.NegativTests</td>
<td></td>
<td>0</td>
<td>com.epam.test.calculator.NegativTests@7ff95560</td></tr>
<tr>
<td title='com.epam.test.calculator.NegativTests.testCtg()'><b>testCtg</b><br>Test class: com.epam.test.calculator.NegativTests</td>
<td></td>
<td>0</td>
<td>com.epam.test.calculator.NegativTests@7ff95560</td></tr>
<tr>
<td title='com.epam.test.calculator.NegativTests.testPow()'><b>testPow</b><br>Test class: com.epam.test.calculator.NegativTests</td>
<td></td>
<td>0</td>
<td>com.epam.test.calculator.NegativTests@7ff95560</td></tr>
<tr>
<td title='com.epam.test.calculator.NegativTests.testTg()'><b>testTg</b><br>Test class: com.epam.test.calculator.NegativTests</td>
<td></td>
<td>0</td>
<td>com.epam.test.calculator.NegativTests@7ff95560</td></tr>
</table><p>
</body>
</html>